Lib/test/test_pyclbr.py

```diff
@@ -214,6 +240,13 @@ def compare(parent1, children1, parent2, children2):
 
         compare(None, actual, None, expected)
 
+    def test_pdb_module(self):
+        with temporary_main_spec():
```

Comment:
With the context manager, it's maybe better to actually put this one back in test_others(). We're also already using `with warnings.catch_warnings()`. Sorry for this oversight!

```py
def test_others(self):
    cm = ...
    ...
    with temporary_main_spec():
        cm('pdb', ...)
```